src/pages/LastComics.tsx

```diff
@@ -1,37 +1,40 @@
-// @ts-nocheck
-import React, { useEffect } from 'react';
-import { useDispatch, useSelector } from 'react-redux';
-import { fetchLastComics } from '../services';
-import { Loader, ComicItem } from '../components';
-import '../styles/LastComics.css';
-
+import React, { useEffect } from "react";
+import { fetchLastComics } from "../services";
+import { Loader, ComicItem } from "../components";
+import "../styles/LastComics.css";
+import { useAppDispatch, useAppSelector } from "../hooks/hooks";
+import { Comic } from "../interfaces/Comic.interface";
 
 function LastComics() {
-    const comics = useSelector(state => state.lastComics.comics);
-    const error = useSelector(state => state.lastComics.error);
-    const loading = useSelector(state => state.lastComics.loading);
-    const dispatch = useDispatch();
-    useEffect(()=>{
-        const getData = async () => {
-            dispatch(fetchLastComics());
-        }
-        getData();
-    },[dispatch])
-    
-    return (
-        <div className="LastComics">
-            {
-            !error && !loading && comics.map((comic) => {
-                return(
-                    <ComicItem  id = {comic.id} coverURL={comic.coverURL} 
-                        title={comic.title} creatorName={comic.creatorName} 
-                        price={comic.price} />
-                    )
-                })
-            }
-            {loading && <Loader/>}
-        </div>
-    )
+  const comics: Comic[] = useAppSelector((state) => state.lastComics.comics);
+  const error = useAppSelector((state) => state.lastComics.error);
+  const loading = useAppSelector((state) => state.lastComics.loading);
+  const dispatch = useAppDispatch();
+  useEffect(() => {
+    const getData = async () => {
+      dispatch(fetchLastComics());
```

Comment:
Can we guarantee that `fetchLastComics` will always be resolved before calling `dispatch`? I believe the `await` keyword is missing.